test/common.py

```diff
@@ -2525,7 +2525,7 @@ def browser_open(cls, url):
         exit_with_error("EMTEST_BROWSER_AUTO_CONFIG only currently works with firefox or chrome.")
       if WINDOWS:
         # Remove directory delimiter backslashes to avoid shlex.split getting confused.
```

Comment:
I guess this comment needs updating.

TBH, I'm not sure `shlex.split` is supposed to be used on windows at all.  But if work then lgtm.